src/main/java/no/skatteetaten/aurora/config/DatabaseConfigDefault.java

```diff
@@ -0,0 +1,85 @@
+package no.skatteetaten.aurora.config;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+import javax.sql.DataSource;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.PropertiesPropertySource;
+
+/**
+ * Configure database if the aurora.db property is set.
+ * This should be moved to a repo of its own.
+ */
+@Configuration
+@ConditionalOnExpression("'${aurora.db:true}' == 'true'")
+@ConditionalOnProperty("db.properties")
+public class DatabaseConfigDefault {
+
+    private static final Logger logger = LoggerFactory.getLogger(DatabaseConfigDefault.class);
+    private ConfigurableEnvironment env;
+
+    public DatabaseConfigDefault(ConfigurableEnvironment env) {
+        this.env = env;
+    }
+
+    @Bean
+    @Profile("openshift")
+    public DataSource dataSource() throws IOException {
```

Comment:
We should probably just set the spring.jdbc properties instead of creating a separate DataSource. This way we will preserve the other spring config features as well.